examples/js_extension/goext.js

```diff
@@ -0,0 +1,184 @@
+// Copyright (C) 2015 NTT Innovation Institute, Inc.
```

Comment:
This is @przemyslaw-dobrowolski-cl 's change. I assume that this is just the example of go extension, so that's why it is called "goext".